brainscore/submission/configuration.py

```diff
@@ -0,0 +1,44 @@
+import datetime
+import distutils
+
+from brainscore.submission.models import Model, Submission
+
+
+class BaseConfig:
+    def __init__(self, work_dir, jenkins_id, db_secret, config_path):
+        self.work_dir = work_dir
+        self.jenkins_id = jenkins_id
+        self.config_path = config_path
+        self.db_secret = db_secret
+        return
+
+
+class MultiConfig(BaseConfig):
+    def __init__(self, model_ids, **kwargs):
+        BaseConfig.__init__(self, **kwargs)
+        self.models = []
+        self.submissions = {}
+        for id in model_ids:
+            model = Model.get(id=id)
+            submission: Submission = model.submission
```

Comment:
ooh fancy, didn't know about this syntax (have only used it in method parameters so far)